com.alientrap.nexuiz-classic.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: com.alientrap.nexuiz-classic
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: launcher.sh
+finish-args:
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --share=network
+  - --persist=.nexuiz
+cleanup:
+- /include
+- /lib/debug
+- /lib/pkgconfig
+- /share/aclocal
+- /share/man
+# windows files
+- /share/nexuiz/*.dll
+- /share/nexuiz/*.exe
+- /share/nexuiz/*.bat
+# mac files
+- /share/nexuiz/Nexuiz-SDL.app
+- /share/nexuiz/Nexuiz.app
+- /share/nexuiz/nexuiz-osx-dedicated
+# extra files (map editor)
+- /share/nexuiz/extra
+# sources and docs
+- /share/nexuiz/sources
+- /share/nexuiz/Docs
+
+modules:
+
+- shared-modules/glu/glu-9.json
+- shared-modules/SDL/SDL-1.2.15.json
+
+- name: nexuiz-classic
+  buildsystem: simple
+  build-commands:
+  - mkdir -p /app/bin/
+  - mkdir -p /app/share/nexuiz/
+  - mv * /app/share/nexuiz/
+  sources:
+  - type: archive
+    url: https://sourceforge.net/projects/nexuiz/files/NexuizRelease/Nexuiz 2.5.2/nexuiz-252.zip
+    sha512: da661eb2ff1764fed1d5f5547b89e4f9aa47613d41a3999c0202d8858ec2b882e4ef0ec182a81d1bdb25dff2ef65aad6af132009884e3557206941a736c2a68c
+
+- name: launcher-i386
```

Comment:
i386 isn't supported anymore starting from 19.08

Reply:
Ah, that's unfortunate, but inevitable.